File: C/tableMultiplication.c
#include <stdio.h>
#include <stdlib.h>


//This is basic algorithm which display the multiplication table of a given integer


int main(int argc, char *argv[])
{
	int nb = 0;
	printf("Ce petit progamme permet d'affichage la table multiplication\nd'un nombre donner en entrée\n");
  	printf("Donner un nombre :\n");
  	scanf("%d", &nb);
  	if(nb < 0)
  	{
    		printf("Erreur - Valeur negitive!\n");
  	}
  	else
  	{
    		printf("La table de multiplication de %d\n", nb);
		for(int i = 1; i <= 10; i++)
		{
      			printf("%d\n", i * nb );
 	 	}

  	}

  return 0;
}

File: Python/tableMultiplication.py
#!/bin/python3
def tableMultiplication():
    nb = input("Donner un nombre: ")
    print("\n")
    nb = int(nb)# Conversion en entier
    i = 0
    if nb < 0:
        print("Erreur - Valeur negative!\n")
    else:
        print("La table de multiplication de ", nb)
        print("\n")
        for i in range(1,11):
            print(i * nb)
            


tableMultiplication()
